projects/Apps/common/src/index.ts

```diff
@@ -218,16 +218,16 @@ export const sizeDescriptions: { [k in ImageSize]: number } = {
     tabletXL: 1140,
 }
 
-export type Edition = string
+export type EditionId = string
 
 // NOTE this list is incomplete and should not be relied on for
 // a complete list of editions - instead use the API/editions endpoint
 export const editions = {
-    daily: 'daily-edition' as Edition,
-    ausWeekly: 'australian-edition' as Edition,
-    usWeekly: 'american-edition' as Edition,
-    dummy: 'the-dummy-edition' as Edition,
-    training: 'training-edition' as Edition,
+    daily: 'daily-edition' as EditionId,
+    ausWeekly: 'australian-edition' as EditionId,
+    usWeekly: 'american-edition' as EditionId,
```

Comment:
should we keep this?

Reply:
I'm not sure - maybe yes to make it slightly easier to add back?!